gateway/constants.py

```diff
@@ -0,0 +1,133 @@
+
+# IBM Cloud and Qiskit
+QISKIT_IBM_URL = "QISKIT_IBM_URL"
+IQP_QCON_API_BASE_URL = "IQP_QCON_API_BASE_URL"
+IAM_IBM_CLOUD_BASE_URL = "IAM_IBM_CLOUD_BASE_URL"
+RESOURCE_CONTROLLER_IBM_CLOUD_BASE_URL = "RESOURCE_CONTROLLER_IBM_CLOUD_BASE_URL"
+RESOURCE_PLANS_ID_ALLOWED = "RESOURCE_PLANS_ID_ALLOWED"
+
+# Custom Image
+CUSTOM_IMAGE_PACKAGE_NAME = "CUSTOM_IMAGE_PACKAGE_NAME"
+CUSTOM_IMAGE_PACKAGE_PATH = "CUSTOM_IMAGE_PACKAGE_PATH"
+
+# Functions
+FUNCTIONS_LOGS_SIZE_LIMIT = "FUNCTIONS_LOGS_SIZE_LIMIT"
+
+# OpenTelemetry
+OTEL_EXPORTER_OTLP_TRACES_ENDPOINT = "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT"
+OTEL_EXPORTER_OTLP_TRACES_INSECURE = "OTEL_EXPORTER_OTLP_TRACES_INSECURE"
+OTEL_ENABLED = "OTEL_ENABLED"
+
+# Default values
+VERSION_DEFAULT = "UNKNOWN"
+DEBUG_DEFAULT = "1"
+DJANGO_SECRET_KEY_DEFAULT = (
+    "django-insecure-&)i3b5aue*#-i6k9i-03qm(d!0h&662lbhj12on_*gimn3x8p7"
+)
+ALLOWED_HOSTS_DEFAULT = "*"
+CSRF_TRUSTED_ORIGINS_DEFAULT = "http://localhost"
+CORS_ALLOWED_ORIGIN_REGEXES_DEFAULT = "http://localhost"
+DATABASE_NAME_DEFAULT = "serverlessdb"
+DATABASE_USER_DEFAULT = "serverlessuser"
+DATABASE_PASSWORD_DEFAULT = "serverlesspassword"
+DATABASE_HOST_DEFAULT = "localhost"
+DATABASE_PORT_DEFAULT = "5432"
+SETTINGS_AUTH_MECHANISM_DEFAULT = "default"
+SETTINGS_AUTH_MOCK_TOKEN_DEFAULT = "awesome_token"
+SITE_HOST_DEFAULT = "http://localhost:8000"
+LIMITS_JOBS_PER_USER_DEFAULT = "2"
+LIMITS_MAX_CLUSTERS_DEFAULT = "6"
+LIMITS_MAX_GPU_CLUSTERS_DEFAULT = "1"
+LIMITS_CPU_PER_TASK_DEFAULT = "4"
+LIMITS_GPU_PER_TASK_DEFAULT = "1"
+LIMITS_MEMORY_PER_TASK_DEFAULT = "8"
+RAY_KUBERAY_NAMESPACE_DEFAULT = "qiskit-serverless"
+RAY_CLUSTER_MODE_LOCAL_DEFAULT = "0"
+RAY_CLUSTER_MODE_LOCAL_HOST_DEFAULT = "http://localhost:8265"
+RAY_NODE_IMAGE_DEFAULT = "icr.io/quantum-public/qiskit-serverless/ray-node:0.27.1"
+RAY_CLUSTER_WORKER_REPLICAS_DEFAULT = "1"
+RAY_CLUSTER_WORKER_REPLICAS_MAX_DEFAULT = "5"
+RAY_CLUSTER_WORKER_MIN_REPLICAS_DEFAULT = "1"
+RAY_CLUSTER_WORKER_MIN_REPLICAS_MAX_DEFAULT = "2"
+RAY_CLUSTER_WORKER_MAX_REPLICAS_DEFAULT = "4"
+RAY_CLUSTER_WORKER_MAX_REPLICAS_MAX_DEFAULT = "10"
+RAY_CLUSTER_MAX_READINESS_TIME_DEFAULT = "120"
+RAY_SETUP_MAX_RETRIES_DEFAULT = "30"
+RAY_CLUSTER_CPU_NODE_SELECTOR_LABEL_DEFAULT = (
+    "ibm-cloud.kubernetes.io/worker-pool-name: default"
+)
+RAY_CLUSTER_GPU_NODE_SELECTOR_LABEL_DEFAULT = (
+    "ibm-cloud.kubernetes.io/worker-pool-name: gpu-workers"
+)
+PROGRAM_TIMEOUT_DEFAULT = "14"
+GATEWAY_ALLOWLIST_CONFIG_DEFAULT = "api/v1/allowlist.json"
+GATEWAY_GPU_JOBS_CONFIG_DEFAULT = "api/v1/gpu-jobs.json"
+GATEWAY_DYNAMIC_DEPENDENCIES_DEFAULT = "requirements-dynamic-dependencies.txt"
+QISKIT_IBM_URL_DEFAULT = "https://cloud.ibm.com"
+CUSTOM_IMAGE_PACKAGE_NAME_DEFAULT = "runner"
+CUSTOM_IMAGE_PACKAGE_PATH_DEFAULT = "/runner"
+FUNCTIONS_LOGS_SIZE_LIMIT_DEFAULT = "50"
+OTEL_EXPORTER_OTLP_TRACES_ENDPOINT_DEFAULT = "http://otel-collector:4317"
+OTEL_EXPORTER_OTLP_TRACES_INSECURE_DEFAULT = "0"
+OTEL_ENABLED_DEFAULT = "0"
```

Comment:
The same approach with the `settings.py` class, why do we create a Config class when we only use it once during bootstrap to assign its values to a bunch of constants that need to be imported? I would remove the `settings.py` and only use the new `Config` class, which should be the de facto source of truth for the config/settings (config = settings, right?).